README.md

```diff
@@ -118,3 +129,33 @@ pip install -e ../harmony-service-lib-py
 Now any changes made to that local repo will be visible in this project when you run tests, etc.
 
 Finally, you can test & run the service in Harmony just as shown in the `Development with Docker` section above.
+
+## Contributions:
+
+Developers working on the NetCDF-to-Zarr service will need to create a feature
+branch for their work. The code in the repository has a `unittest` suite, which
+should be updated when any code is added or updated within the repository.
+
+When a feature branch is ready for review, a Pull Request (PR) should be opened
+against the `main` branch. This will automatically trigger a GitHub workflow
+that will run the `unittest` suite.
+
+When a PR is merged against the `main` branch, a different workflow will check
```

Comment:
Can you name explicitly which github actions will run here?  and below next paragraph?